cypress/integration/spec.js

```diff
@@ -54,7 +64,9 @@ describe('Chart Explorer', function() {
     ).click()
 
     cy.get('#react-select-2-option-3').click()
```

Comment:
it feels strange to me to end a test with a `click`, and not have an assertion after. is there some sort of specific behavior you're trying to trigger by performing this click? if so, it seems like it would be appropriate to assert that said behavior has happened.

Reply:
hmm, so this whole beforeEach function just sets up chart-builder with some basic chart configuration so that the later tests can perform actions like downloading, sharing or editing the chart. 

I'm not sure if theres a better approach to doing something like this here❓

Reply:
ah ok ok, I didn't notice that this was part of a `before`. Makes sense now!